airflow/ui/package.json

```diff
@@ -27,6 +28,8 @@
     "@visx/group": "^3.12.0",
     "@visx/shape": "^3.12.0",
     "@xyflow/react": "^12.3.5",
+    "accordion": "^3.0.2",
```

Comment:
Interesting... why is accordion being added here? An accordion is already using in the TriggerDAGForm - can we consolidate and not use two libs in two areas?

Reply:
Good catch. Yes, let's use chakra's accordion

Reply:
I think I got confused, should be better now. ty